Base/QTCore/qSlicerExtensionsManagerModel.h

```diff
@@ -399,6 +399,17 @@ public slots:
   /// Returns false if waitForCompletion is set to true and metadata cannot be retrieved.
   bool updateExtensionsMetadataFromServer(bool force=false, bool waitForCompletion=false);
 
+  /// \brief Enable/disable the use of limit and offset parameters for querying extension metadata from the extensions server.
+  ///
+  /// If enabled (the default), the metadata are retrieved by batch of at most 50 extensions until
```

Comment:
If enabled (the default), the server may return metadata for a limited number of extensions at once (for example, 50) to keep the server's load and response time low. The model automatically submits additional queries until all data is retrieved.